README.md

```diff
@@ -1,45 +1,112 @@
+[![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg?style=flat-square)](http://makeapullrequest.com)
+
 # Football Players Statistics WebCrawler
 
 This project is a sub-module for [Multiplayer Football Draft Simulator](https://github.com/sauravhiremath/fifa).
 
 # About
 
+[![forthebadge made-with-python](http://ForTheBadge.com/images/badges/made-with-python.svg)](https://www.python.org/)
+
 A web-crawler to scrape all football players' information from [Sofifa](https://sofifa.com/players) and exporting it to JSON format. Perform data cleaning and analytics on the obtained data
 
-* Crawler: Built on scrapy using python3
-* Analytics: IPynb noteboook python3
+- Crawler: Built on scrapy using python3
+- Analytics: IPynb noteboook python3
 
 Further exported to the [Football Draft Backend](https://github.com/sauravhiremath/fifa-api) to serve from an endpoint
 
 # Steps to run the project
 
-* Install project dependencies <br>
-    ```python
-    pip install -r requirements.txt
-    ```
+- Install project dependencies <br>
 
-* Run the crawler with _./fifa-crawler_ as current directory (This the main scrapy crawler directory)
-    > Make sure to change the filenames to read and write appropriately: <br>
-    > `players_url.json` --> scraping urls <br>
-    > `players_stats_raw.json` --> scraping player stats
+  ```python
+  pip install -r requirements.txt
+  ```
 
-    * First run the URL spider (To get all players urls)
-        ```python
-        scrapy crawl players_url
-        ```
-    * After successfull, run the stats spider (To get the players statistics from URLs from above)
-        ```python
-        scrapy crawl players_stats
-        ```
+- Run the crawler with _./fifa-crawler_ as current directory (This the main scrapy crawler directory)
 
+  > Make sure to change the filenames to read and write appropriately: <br> > `players_url.json` --> scraping urls <br> > `players_stats_raw.json` --> scraping player stats
```

Comment:
This line seems to have a weird rendering. Linebreaks (Unix style `\n`) should solve the issue.

![image](https://user-images.githubusercontent.com/28642011/94953167-b33c1780-0504-11eb-8608-7570afb2255a.png)

Reply:
Fixed in the new commit!